### DISCUSSION TITLE
Geofencing events not triggering Android issue

### DISCUSSION THREAD

**Discussion by dinonovak** [shinyorg/shiny#1112] [Q&A]
I am trying to use geofencing (only) on Android.
I am using Geofencing sample. Shiny 2.5.3 (Notifications & Locations) , Forms 5.0.0.241

When I set GPX route for playback in simulator (Api_30), despite geofence being on the route, onEnter / onExit are not triggered.
If I put open and put Google maps app in the foreground, so that location is tracked on map, geofences are triggered.

What could be the issue here?

**Comment by dinonovak** [comment]
Found this link, could it really be the case that Google has disabled it now?
https://stackoverflow.com/questions/60406684/android-geofencing-only-works-when-google-maps-is-running-on-any-app

**Reply by aritchie** [reply]
Yes, geofencing is known to be an issue for many simulator setups on Android.  You could be missing google play services.  There are many number of things.  Test on a device with mock setups.  

Please note: if you do have an issue - this is NOT where you file it.  You must be using the latest version (2.6 also contains most of the android 12 permission fixes) and file it here with a reproducible sample: https://github.com/shinyorg/shiny/issues/new/choose
